crypto/src/signature.rs

```diff
@@ -51,7 +51,7 @@ pub type Payload = Vec<u8>;
 pub struct Signature {
     /// Public key that is used for verification. Payload is verified by algorithm
     /// that corresponds with the public key's digest function.
-    public_key: PublicKey,
+    pub public_key: PublicKey,
```

Comment:
please restore. This is part of the public facing API and we don't want to be exposing internals of our structures to the public. If you think this forces you to do much cloning in the `iroha_core` we should introduce `transparent_api` as we're going to do in #3127

Reply:
I didn't realize that the derived Getter trait added a getter. Will revert this change.